bin/ghe-backup

```diff
@@ -65,7 +65,7 @@ if [ -f ../in-progress ]; then
   progress=$(cat ../in-progress)
   snapshot=$(echo "$progress" | cut -d ' ' -f 1)
   pid=$(echo "$progress" | cut -d ' ' -f 2)
-  if ! ps -p $pid -o command= | grep ghe-backup; then
+  if ! ps -p $pid -o comm= | grep ghe-backup; then
```

Comment:
@janorn this change will break backup-utils in Linux hosts unfortunately, it's not equivalent:

```
rubiojr@enterprise2-dev5-cp1-prd:~/backup-utils$ ps -p 103090 -o comm=
bash
rubiojr@enterprise2-dev5-cp1-prd:~/backup-utils$ ps -p 103090 -o command=
bash ./bin/ghe-backup
```